mesa/space.py

```diff
@@ -666,10 +666,13 @@ def find_neighbors(pos: Coordinate, radius: int) -> None:
             if radius > 1:
                 [find_neighbors(coords, radius - 1) for coords in adjacent]
 
+        explored, coordinates = dict(), set()
```

Comment:
They should have been defined in L636. And should have been split in 2 lines.

Reply:
done!